flutter/test/sentry_navigator_observer_test.dart

```diff
@@ -259,46 +263,40 @@ void main() {
       });
     });
 
-    test('do not bind to scope if already set', () {
```

Comment:
we can re-add this test, right?

Reply:
yea, now that we only remove the scope span if it's the transaction from navigator observer this test is again useful